include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -816,11 +837,22 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", []> {
         for(int k = 0; k < 8 /*size_0*/; k++)
           // access/store element at/to index (i * 16 /*stride_2*/ + j * 1 /*stride_1*/ + k * 2 /*stride_0*/)
     ```
+
+    ## Important gotcha regarding strides
+
+    All strides are expressed in multiples of the element width (just like `len` and `offset`)
+    **with the caveat that the inner-most dimension's stride must be a multiple of 32b**.
```

Comment:
I know what you're trying to say, but this seems like its saying "Exists some k s.t. stride = k * 32b"   How about we just say "Addresses must be 32bit aligned"?

Reply:
This is wrong now - last stride must be 1.